dali/pipeline/operators/geometric/bb_flip_test.cc

```diff
@@ -124,6 +121,18 @@ std::vector<Roi> FromTensorWrapper(TensorListWrapper tw) {
   return ret;
 }
 
+template<typename Backend>
+std::vector<Roi> FromTensorWrapper(TensorListWrapper tw) {
```

Comment:
`FromTensorListPtr` and `FromTensorWrapper` can be overloads of `FromTensor(...)`

Reply:
That would suggest the same level of error checking on both.